charts/openmetadata/values.yaml

```
@@ -389,5 +389,11 @@ startupProbe:
     path: /healthcheck
     port: http-admin
 
+podDisruptionBudget:
+  enabled: true
```

By default, it should be enabled.
```suggestion
  enabled: false
```

---

Changing the code to have PDB disabled by default.